packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1021,63 +1021,6 @@ class EditableText extends StatefulWidget {
   /// Called when the user indicates that they are done editing the text in the
   /// field.
   /// {@endtemplate}
-  ///
```

Comment:
I tried to keep the example by changing it to "moving focus to a specific input field" but found it not as common/practical. @gspencergoog any suggestions?

Reply:
I think it's fine to remove this if it isn't relevant anymore.